pyemd.c

```diff
@@ -120,8 +130,20 @@ static PyObject *compute_emd(PyObject *self, PyObject *args, PyObject *keywds)
   signature2.Features = f2;
   signature2.Weights = w2;
 
-  distance = emd(&signature1, &signature2, double_dist, 0, 0);
+  distance = emd(&signature1, &signature2, double_dist, flow, &flow_size);
 
+  if (return_flows){
+	  PyObject *returnList = PyList_New((Py_ssize_t) flow_size);
+	  for(i = 0; i < flow_size; i++) {
+		  PyObject *tuple = PyTuple_New(3);
+		  PyTuple_SetItem(tuple, 0, Py_BuildValue("I", flow[i].from));
+		  PyTuple_SetItem(tuple, 1, Py_BuildValue("I", flow[i].to));
+		  PyTuple_SetItem(tuple, 2, Py_BuildValue("d", flow[i].amount));
+		  PyList_SetItem(returnList, i, tuple);
+	  }
+	  return returnList;
```

Comment:
Would it be possible to return a tuple of (distance, flows) rather than just the flows?